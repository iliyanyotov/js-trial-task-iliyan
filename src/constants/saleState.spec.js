import SALE_STATE from './saleState';

describe('constants: SALE_STATE', () => {
  it('should match the snapshot', () => {
    expect(SALE_STATE).toMatchSnapshot();
  });
});
