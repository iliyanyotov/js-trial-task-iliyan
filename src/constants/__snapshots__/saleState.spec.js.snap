// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants: SALE_STATE should match the snapshot 1`] = `
Object {
  "FAILED": "failed",
  "REFUNDED": "refunded",
  "SUCCESSFUL": "successful",
}
`;
